examples/blas/run_time_dispatching/level3/gemm_usm.cpp

```diff
@@ -0,0 +1,247 @@
+/*******************************************************************************
+* Copyright 2020-2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions
+* and limitations under the License.
+*
+*
+* SPDX-License-Identifier: Apache-2.0
+*******************************************************************************/
+
+/*
+*
+*  Content:
+*       This example demonstrates use of DPCPP API oneapi::mkl::blas::gemm
+*       using unified shared memory to perform General Matrix-Matrix
+*       Multiplication on a SYCL device (HOST, CPU, GPU) that is selected
```

Comment:
Is HOST device supported?

Reply:
yes, use `export SYCL_DEVICE_ILTER=host`

Reply:
I meant, I don't think SYCL supports HOST device officially.

Reply:
In https://intel.github.io/llvm-docs/EnvironmentVariables.html#sycl-device-filter we have host as a possible value for backends and it seems that host should work when specified explicitly.
`The RT does not include the host backend and the host device automatically unless one of the filters explicitly specifies the host device type. Therefore, SYCL_DEVICE_FILTER=host should be set to enforce SYCL to use the host device only.`
That's why I included it as an option here for users. But in ctest we don't test with host